sources/cfsignal/src/cloudformation.rs

```diff
@@ -21,7 +22,7 @@ pub async fn signal_resource(
         "Region: {:?} - InstanceID: {:?} - Signal: {:?}",
         region, instance_id, status
     );
-    let config = aws_config::from_env()
+    let config = aws_config::defaults(BehaviorVersion::v2023_11_09())
```

Comment:
Do we have a reason to lock to this version over latest?

Reply:
Yes this is to prevent any behavior breakage should the GA sdk change the behaviors there-in on a version tick